service/raft/raft_group0.cc

```diff
@@ -206,15 +207,16 @@ raft_server_for_group raft_group0::create_server_for_group0(raft::group_id gid,
 }
 
 future<group0_info>
-raft_group0::discover_group0(raft::server_address my_addr, const std::vector<raft::server_info>& seed_infos) {
+raft_group0::discover_group0(raft::server_id my_id, const std::vector<raft::server_info>& seed_infos) {
     std::vector<raft::server_address> seeds;
     for (auto& info : seed_infos) {
         seeds.emplace_back(raft::server_id{}, info);
     }
+    raft::server_address self{my_id, inet_addr_to_raft_addr(_gossiper.get_broadcast_address())};
```

Comment:
Why the rename from `my_addr` to `self`? Note that `self` is also used to name `raft_group0&` below.

Reply:
I renamed it back.